cmd/piper.go

```diff
@@ -32,6 +32,8 @@ type GeneralConfigOptions struct {
 	StepName             string
 	Verbose              bool
 	LogFormat            string
+	VaultApproleID       string
```

Comment:
What is an *Approle*? Would it be sufficient to name it `VaultID`?

Reply:
An AppRole is one of many ways to authenticate against vault. We could shorten it to VaultRoleID if it's too long. `VaultApproleID` is already short for `VaultApproleCredentialID` :)

Reply:
https://www.vaultproject.io/docs/auth/approle